dev/devicelab/bin/tasks/technical_debt__cost.dart

```diff
@@ -137,6 +137,7 @@ Future<void> main() async {
         _kCostBenchmarkKey,
         _kNumberOfDependenciesKey,
         _kNumberOfConsumerDependenciesKey,
+        _kNumberOfFlutterToolGlobals,
```

Comment:
I forgot to add this key, so the benchmark isn't showing up 😢

Reply:
Do you want to land adding the key in a separate PR to get initial numbers first before landing this?

Reply:
Good idea, opened here: https://github.com/flutter/flutter/pull/48266

Reply:
Fixed